src/component/page/deltakerliste-veileder/ingen-deltakere/IngenDeltakere.module.scss

```diff
@@ -1,12 +1,9 @@
 .ingenDeltakere {
```

Comment:
Vi må ha en media-query her for at ingendeltaker boksen ikke skal legge seg "oppå" andre elementer på mindre skjermer.
```css
.ingenDeltakere {
  padding: 2rem;
  max-width: 600px;
  @media(min-width: 900px) {
      position: absolute;
      top: 33%;
      left: 50%;
      transform: translate(-50%, -50%);
  }
}
```

Reply:
I fix :)